File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/VeilBreaker.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class VeilBreaker extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_veil_breaker_name;
        this.idDescription = R.string.item_veil_breaker_description;
        this.idImage = R.drawable.veil_breaker;
        this.source.add(Integer.valueOf(R.string.dungeon_name_hidden_city_of_larox));
        this.price = 20L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/VeilBreaker.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class VeilBreaker : Item() {
    override fun configureProperties() {
        idName = R.string.item_veil_breaker_name
        idDescription = R.string.item_veil_breaker_description
        idImage = R.drawable.veil_breaker
        source.add(R.string.dungeon_name_hidden_city_of_larox)
        price = 20L
    }
}
